env/production/alarms/terragrunt.hcl

```diff
@@ -60,6 +60,20 @@ dependency "app" {
   }
 }
 
+dependency "sns" {
```

Comment:
You'll also have to add the `sns` dependency to the `dependencies` block at the top of the file.  This lets Terragrunt know the order that dependencies should be applied if you use a command like `terragrunt run-all plan`:
```hcl
dependencies {
  paths = ["../hosted_zone", "../kms", "../load_balancer", "../sqs", "../app", "../sns"]
}
```